tools/db/move-team/src/Work.hs

```diff
@@ -0,0 +1,183 @@
+assertTargetDirEmpty Env {..} = do
+  files <- listDirectory envTargetPath
+  unless (Imports.null files) $ do
+    IO.putStrLn $ "Target directory " <> envTargetPath <> " is not empty."
+    exitWith (ExitFailure 1)
+
+runExport :: Env -> IO ()
+runExport env@Env {..} = do
+  ExitSuccess <- system $ "mkdir -p " <> show envTargetPath
+  assertTargetDirEmpty env
+  runTeam env
+  runGalleyTeamMembers env
+  runGalleyTeamConv env
+  readAllTeamUserIds env >>= runFullScans env
+
+runImport :: Env -> IO ()
+runImport env = do
+  IO.putStrLn "runImport"
+  importAllTables env
+
+runDebugExportFull :: Env -> IO ()
+runDebugExportFull env = do
+  assertTargetDirEmpty env
+  exportAllTablesFull env
+
+----------------------------------------------------------------------
+
+runTeam :: Env -> IO ()
+runTeam env@Env {..} = do
+  appendJsonLines (envTargetPath </> "galley.billing_team_member") (readGalleyBillingTeamMember env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team") (readGalleyTeam env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team_features") (readGalleyTeamFeatures env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team_member") (readGalleyTeamMember env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team_notifications") (readGalleyTeamNotifications env envTeamId)
+
+runGalleyTeamMembers :: Env -> IO ()
+runGalleyTeamMembers env@Env {..} =
+  IO.withBinaryFile (envTargetPath </> "galley.team_member") IO.WriteMode $ \outH ->
+    runConduit $
+      zipSources
+        (CL.sourceList [(1 :: Int32) ..])
+        (readGalleyTeamMember env envTeamId)
+        .| C.mapM (handleTeamMembers env)
+        .| sinkJsonLines outH
+
+handleTeamMembers :: Env -> (Int32, [RowGalleyTeamMember]) -> IO [RowGalleyTeamMember]
+handleTeamMembers env@Env {..} (i, members) = do
+  Log.info envLogger (Log.field "number of team members loaded: " (show (i * envPageSize)))
+  let uids = catMaybes $ fmap Id . view _2 <$> members
+
+  appendJsonLines (envTargetPath </> "brig.clients") (readBrigClients env uids)
+  appendJsonLines (envTargetPath </> "brig.connection") (readBrigConnection env uids)
+  appendJsonLines (envTargetPath </> "brig.connection") (readBrigIdMapping env uids)
+  appendJsonLines (envTargetPath </> "brig.login_codes") (readBrigLoginCodes env uids)
+  appendJsonLines (envTargetPath </> "brig.prekeys") (readBrigPrekeys env uids)
+  appendJsonLines (envTargetPath </> "brig.properties") (readBrigProperties env uids)
+  appendJsonLines (envTargetPath </> "brig.rich_info") (readBrigRichInfo env uids)
+  appendJsonLines (envTargetPath </> "brig.user") (readBrigUser env uids)
+  appendJsonLines (envTargetPath </> "galley.clients") (readGalleyClients env uids)
+  appendJsonLines (envTargetPath </> "galley.user") (readGalleyUser env uids)
+  appendJsonLines (envTargetPath </> "galley.user_team") (readGalleyUserTeam env uids)
+  appendJsonLines (envTargetPath </> "gundeck.notifications") (readGalleyUserTeam env uids)
```

Comment:
```suggestion
  appendJsonLines (envTargetPath </> "gundeck.notifications") (readGalleyNotifications env uids)
```

Reply:
Ouch again... Was able to reproduce errors when this is not fixed (see below).